### PR TITLE
fix: use redirectFrom to navigate after logging in from top menu

### PR DESCRIPTION
**Pull request by brandonroberts**
## Description

- Navigates back to the provided `redirectedFrom` after authenticating through the "Connect to GitHub" button

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Review by nickytonline — APPROVED** [review]
:shipit:

File: components/molecules/AuthSection/auth-section.tsx [hunk 1]
```diff
@@ -42,6 +43,12 @@ const AuthSection: React.FC = ({}) => {
     }
   }, []);
 
+  useEffect(() => {
```
Comment (nickytonline) [inline components/molecules/AuthSection/auth-section.tsx:46]: Jsut wondering if in a future PR we can handle the redirect server side?

Comment (brandonroberts) [reply components/molecules/AuthSection/auth-section.tsx:46]: Yep. I chose not to mix it in with the workspaces work for now

**Review by zeucapua — APPROVED** [review]
LGTM

**Pull request merged**